packages/react/src/Form/Radio/Client/RadioItem.tsx

```diff
@@ -1,60 +1,87 @@
 import "@axa-fr/design-system-css/dist/Form/Radio/Client/Radio.scss";
 import { ComponentPropsWithRef, forwardRef, ReactNode, useId } from "react";
-import { getOptionClassName } from "../../core";
 
-type Props = Omit<ComponentPropsWithRef<"input">, "checked" | "type"> & {
-  classModifier?: string;
-  optionClassName?: string;
+export type RadioItemOption = {
+  id?: string;
   icon?: ReactNode;
-  choice?: ReactNode;
+  title: ReactNode;
+  subtitle?: ReactNode;
   description?: ReactNode;
-  isChecked?: boolean;
+  value: string;
+  disabled?: boolean;
+  erroneous?: boolean;
+  centerContent?: boolean;
 };
 
+type Props = Omit<ComponentPropsWithRef<"input">, "type" | "title"> &
+  RadioItemOption & {
+    checked?: boolean;
+  };
+
 const RadioItem = forwardRef<HTMLInputElement, Props>(
   (
     {
+      disabled = false,
+      erroneous = false,
+      centerContent = false,
       value = "",
       id,
-      isChecked,
-      children,
       icon,
-      choice,
+      title,
+      subtitle,
       description,
-      classModifier = "",
-      className = "",
-      disabled = false,
+      className,
+      checked,
```

Comment:
`checked` can be in `...otherProps` no ?

Reply:
That's correct. 
Fixed !